src/ngraph/frontend/onnx_import/op/reduce.hpp

```diff
@@ -98,8 +99,12 @@ namespace ngraph
                 ///
                 inline NodeVector reduce_l1(const Node& node)
                 {
+                    auto reduce_function = std::bind(ngraph::builder::l1_norm,
```

Comment:
Please change the name from generic `reduce_function` to specific, like: `l1_norm_reduction`

Reply:
Doesn't this come from the name of the function this is in?